deployment/terraform_azure/aks.tf

```diff
@@ -73,3 +66,20 @@ resource "local_file" "aks_config" {
   filename = "aks_kubectl_config"
   content  = azurerm_kubernetes_cluster.default.kube_config_raw
 }
+
+resource "kubernetes_namespace" "eventuate" {
+  metadata {
+    name = "eventuate-tram-examples-customers-and-orders"
+  }
+}
+
+resource "kubernetes_config_map" "mongo" {
+  metadata {
```

Comment:
There needs to a YAML defining the config map for the non-AZ case.

Reply:
Added config map file - deployment/kubernetes/secrets/mongodb-config.yaml
This needs to be deployed only on local k8s cluster.